docs/index.rst

```diff
@@ -1,380 +1,150 @@
+   whenever a unexpected token was found. In addition it will print all the
+   issues found for each test and also a summary with how many tests have
+   failed each of the checks.
+
+Testimony can also print colored output, make sure ``termcolor`` package is
+installed and the ``--no-color`` option was not specified.
 
 Advantages
 ----------
-1. Avoids a separate test case management tool to document test cases by using the python automation framework for the same
+1. Avoids a separate test case management tool to document test cases by using
+   the Python automation framework for the same
 2. Enforces test cases to follow a defined standard
-3. Runs in integration with tools like Travis to get a report as and when you check in code
-4. Saves a lot of time from the conventional way of writing test cases using a test management tool
-5. The test case information can be easily extracted from testimony and can be ported to any test management tool 
+3. Runs in integration with tools like Travis to get a report as and when you
+   check in code
+4. Saves a lot of time from the conventional way of writing test cases using a
+   test management tool
+5. The test case information can be easily extracted from Testimony and can be
+   ported to any test management tool
 
 Installation
 ------------
 
-You could install testimony from PyPI using pip:
-
-::
+You can install Testimony from PyPI using pip::
 
     pip install testimony
 
-Requirements
-------------
-\1) Install testimony following the steps in the installation section
-
-\2) Pre-requisites
-
-Expected Docstring format::
-
-    """Log in as a valid user
-
-    @Feature: Login
-
-    @Setup: Navigate to abc.com
-
-    @Steps:
-
-    1. Launch the url
-    2. Log in with valid user credentials
-
-    @Assert: Log in successful
-
-    @BZ: #1234567
-
-    @Status: Manual (REMOVE this field once automated)
-
-    @Tags: T1, T2, T3
-
-    """
-
-    Note: The first line of docstring will be considered as the test case name.
-
-\3) Optional color formatting - If termcolor package is installed, output will be printed in colored text
-
-Usage:
-------
-
-help::
-
-    $ testimony -h
-    usage: testimony [-h] [-n] [-t TAGS] REPORT PATH [PATH ...]
-
-    Inspects and report on the Python test cases.
-
-    positional arguments:
-  	REPORT                report type, possible values: print, summary,
-    	                  validate_docstring, bugs, manual, auto, tags
-  	PATH                  a list of paths to look for tests cases
-
-  	optional arguments:
-  	-h, --help            show this help message and exit
-  	-j, --json            JSON output
-  	-n, --nocolor         Do not use color option
-	-t [TAGS [TAGS ...]], --tags [TAGS [TAGS ...]]
-                          space separated tags to search.  Note: Always run this
-                          only in the root of the project where test cases are
-                          stored
-	
-print - test cases::
```

Comment:
Removed the outputs from the documentation since it may change and the documentation do not match the current output.